### PR TITLE
feat: add cookie banner on docs [sc-19436]

### PR DESCRIPTION
**Pull request by pilimartinez**
## Affected Components
* [ ] Content & Marketing
* [ ] Pricing
* [ ] Test
* [X] Docs
* [X] Learn
* [X] Other

## Pre-Requisites
* [ ] Code is linted (`npm run lint`)

## Notes for the Reviewer

**We can't test this before merging due to CookieYes limitations**

Added the CookieYes script to include the cookie banner on the repository.

### CODE REVIEW

**Review by tnolet — COMMENTED** [review]
I don't understand. This has no effect on the code that is actually setting cookies. e.g.  https://github.com/checkly/docs.checklyhq.com/blob/b273270a8d4ecdd8f2b6d7f2b4612be6f71da796/site/layouts/partials/head.html#L66

**Comment by pilimartinez** [comment]
> I don't understand. This has no effect on the code that is actually setting cookies. e.g.
> 
> https://github.com/checkly/docs.checklyhq.com/blob/b273270a8d4ecdd8f2b6d7f2b4612be6f71da796/site/layouts/partials/head.html#L66

I thought we were setting the cookies with that cookie script. Are they coming from gtm?

**Comment by tnolet** [comment]
> > I don't understand. This has no effect on the code that is actually setting cookies. e.g.
> > https://github.com/checkly/docs.checklyhq.com/blob/b273270a8d4ecdd8f2b6d7f2b4612be6f71da796/site/layouts/partials/head.html#L66
> 
> I thought we were setting the cookies with that cookie script. Are they coming from gtm?

We are adding 3rd party libraries at the bottom of the `head.html` partial. That will still happen, even with a new banner.

**Comment by pilimartinez** [comment]
> > > I don't understand. This has no effect on the code that is actually setting cookies. e.g.
> > > https://github.com/checkly/docs.checklyhq.com/blob/b273270a8d4ecdd8f2b6d7f2b4612be6f71da796/site/layouts/partials/head.html#L66
> > 
> > 
> > I thought we were setting the cookies with that cookie script. Are they coming from gtm?
> 
> We are adding 3rd party libraries at the bottom of the `head.html` partial. That will still happen, even with a new banner.

CookieYes has a blocking mechanism that should take care of blocking that before consent. I also added a block code to the scrips as suggested [here](https://www.cookieyes.com/documentation/implement-prior-consent-using-cookieyes/#subhead2-0)

Captura de Pantalla 2024-02-26 a la(s) 14 55 20

File: site/layouts/partials/head.html [hunk 1]
```diff
@@ -31,7 +31,7 @@
   <link rel="canonical" href="{{ .Permalink }}" />
 
 
-  <script async src="https://cdn.cookielaw.org/consent/e4c8a868-c0ad-4c0c-b61e-f6eb6cad2810.js" type="text/javascript" charset="UTF-8"></script>
+	<script id="cookieyes" type="text/javascript" src="https://cdn-cookieyes.com/client_data/2853758a17c7c3979d959d32/script.js"></script>
   <script type="text/javascript">
```
[the remark this answers is not in the document]
Comment (pilimartinez) [reply site/layouts/partials/head.html]: Should be removed now 👍

**Pull request merged**